app/database/schema/server/table_schemas/user.ts

```diff
@@ -28,5 +28,6 @@ export default tableSchema({
         {name: 'status', type: 'string'},
         {name: 'timezone', type: 'string'},
         {name: 'username', type: 'string'},
+        {name: 'remote_id', type: 'string'},
```

Comment:
in my opinion, as this field will be rarely set I would rather make this field optional https://nozbe.github.io/WatermelonDB/Schema.html because we will fill the database and each object that we query with an empty array almost at all times, that takes about 1 byte per object without need.